### PR TITLE
Remove installation scheme tweaking hack.

### PR DESCRIPTION
**Pull request by jpakkane**
It was needed on older Debians.

It breaks new Debian.

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1020567

### CODE REVIEW

**Comment by eli-schwartz** [comment]
Really, astoundingly, amazing.

Debian patches to Python mean that some things are mandatory on some versions of Debian, and a fatal error on other versions of Debian.

...

What is upstream Debian's recommendation? Should software authors encode a version table of python build ids and do different things for each build id whenever Debian is detected? @doko42

File: run_meson_command_tests.py [hunk 1]
```diff
@@ -31,8 +31,6 @@
     scheme = sysconfig.get_default_scheme()
 else:
     scheme = sysconfig._get_default_scheme()
-if scheme == 'posix_local':
-    scheme = 'posix_prefix'
```
Comment (eli-schwartz) [inline run_meson_command_tests.py:35]: This still leaves quite a bit of pointless code floating around here. We should simply do a straight `git revert a2079939de2d1d271e5ea7f23c3656a73e325728` of #10297

**Review by eli-schwartz — CHANGES_REQUESTED** [review]

File: run_meson_command_tests.py (hunk 1 above)
Comment (jpakkane) [reply run_meson_command_tests.py:35]: ¨FTR I tested that and things still work.

**Comment by eli-schwartz** [comment]
This turns out to be broken on Ubuntu rolling, so I'll reiterate:

> What is upstream Debian's recommendation? Should software authors encode a version table of python build ids and do different things for each build id whenever Debian is detected?

**Comment by jpakkane** [comment]
Windows failures are due to failing to upload coverage results. All tests did pass. Not sure what's up with Cygwin, though.

OTOH this revealed that the command tests are not actually run, because this version has a very clear bug. It should not have passed. Deciphering...

**Comment by jpakkane** [comment]
The version I pushed now works when applied on the .63 branch but fails for master. I don't know why. Unless someone comes up with a proper fix, I'd say we get this in so we can get the point release out and fix master releases if and when it becomes an issue.

**Comment by jpakkane** [comment]
Cygwin failure is unrelated, merging as per discussion (and because this is already in the point release).

**Pull request merged**